### PR TITLE
feat: add wizard e2e test query edge case

### PR DESCRIPTION
**Pull request by jonathanlab**
## Problem

To support the fixture generation during E2E tests in the `@posthog/wizard` package, we need to support mocking the cache value, since we never authenticate through the browser in the E2E flow.

Context: https://github.com/PostHog/wizard/pull/101

## Changes

- Add the ability to mock wizard data if PostHog is running in debug and the `X-PostHog-Wizard-Fixture-Generation` header is specified.
- Upped the Wizard rate limit in development.

## How did you test this code?

Added tests for checking the changed functionality.

## Did you write or update any docs for this change?

- [x] No docs needed for this change

---

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: posthog/rate_limit.py [hunk 1]
```diff
@@ -427,6 +427,9 @@ class SetupWizardAuthenticationRateThrottle(UserRateThrottle):
 class SetupWizardQueryRateThrottle(SimpleRateThrottle):
     rate = "20/day"  # Since the authentication hash is valid for a short period, this is effectively per-user
 
+    if settings.DEBUG:
+        "1000/day"
```
[the remark this answers is not in the document]
Comment (jonathanlab) [reply posthog/rate_limit.py]: <3

**Review by joshsny — APPROVED** [review]
🚢
